packages/app-form-builder/src/admin/components/FormEditor/Droppable.tsx

```diff
@@ -24,13 +24,13 @@ export interface IsVisibleCallableParams {
     type: string;
     isDragging: boolean;
     ui: string;
-    pos?: Partial<FieldLayoutPositionType>;
+    pos?: Partial<FieldLayoutPositionType> | Record<any, any>;
 }
 export interface IsVisibleCallable {
     (params: IsVisibleCallableParams): boolean;
 }
 export interface OnDropCallable {
-    (item: DragObjectWithType): DroppableDropResult | undefined;
+    (item: DragObjectWithType & { ui: string }): DroppableDropResult | undefined;
```

Comment:
We need that `{ ui: string }` because `DragObjectWithType` is the type from the library and it does not have fields that we set through `beginDrag`. I will probably move `DragObjectWithType & { ui: string }` into separate type
https://github.com/webiny/webiny-js/pull/3436/files#diff-33b595e920d5882dbdc98d6382274a874914bbfc92ff1b4c27df2ff8a711e48cR206